AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -961,7 +961,7 @@ public void run() {
    * @param response The response containing content.
    */
   private static void applyContentInResponse(JSONObject response) {
-    Map<String, Object> values = JsonConverter.mapFromJsonOrDefault(
+    Map<String, Object> values = JsonConverter.mapFromJson(
```

Comment:
Following line 794 null `response.optJSONObject(Constants.Keys.VARS)` is an backend error, but `JsonConverter.mapFromJsonOrDefault` return emptyMap in this case.

`JsonConverter.mapFromJson` will return null and variables will not be changed at all (previous values will be used)